conf/messages.cy

```diff
@@ -717,19 +717,11 @@ valuationForTaxes.phone2 = Rhif f ffôn (Cymru) 03000 505505
 valuationForTaxes.openingHours-title = Oriau agor
 valuationForTaxes.openingHours =  08:30 – 17:00. Ar gau ar wyliau’r banc.
  
-providingLettings.title = Manylion cyswllt ar gyfer y Swyddogion Rhent - Ffurflen Gysylltu Asiantaeth y Swyddfa Brisio - GOV.UK
-providingLettings.heading = Manylion cyswllt ar gyfer y Swyddogion Rhent
-providingLettings.address.heading = Swyddogion Rhent Asiantaeth y Swyddfa Brisio: cyhoeddus
-providingLettings.address.1 = Swyddfa Cymorth Rhwydwaith - Lwfansau Tai
-providingLettings.address.2 = Llys Anwyl
-providingLettings.address.3 = Ffordd Churton
-providingLettings.address.4 = Y Rhyl
-providingLettings.address.5 = LL18 3NB
-providingLettings.email-title = E-bost:
-providingLettings.email = NSOhelpdesk@voa.gov.uk
-providingLettings.phone-title =  Ffôn
-providingLettings.phone-number = 03000 502502
-
+providingLettings.title = Darparwch wybodaeth i’r Swyddogion Rhent ar-lein - Valuation Office Agency contact form - GOV.UK
```

Comment:
Is this correct - title has English. As per the confluence page it should be Darparwch wybodaeth i’r Swyddogion Rhent ar-lein - Ffurflen Gysylltu Asiantaeth y Swyddfa Brisio - GOV.UK
@evanspete

Reply:
Changes have been made and committed in a new push!